tests/StarterTheme/specs/opc.js

```diff
@@ -12,6 +12,72 @@ function toggleAllTermsCheckboxes () {
 }
 
 describe('The One Page Checkout', function () {
+
+  before(function () {
+    return browser.url('/').click('.menu a').click('a[data-link-action="add-to-cart"]');
```

Comment:
Please add a short pause after clicking the add-to-cart link: since adding to the cart is done in AJAX, sometimes when the actual test starts I don't have a product in the cart yet :)